qt/SessionDialog.cc

```diff
@@ -34,17 +30,13 @@ SessionDialog::resensitize ()
   const bool isRemote = ui.remoteSessionRadio->isChecked ();
   const bool useAuth = ui.authCheck->isChecked ();
 
-  for (QWidget * const w: myRemoteWidgets)
+  foreach (QWidget * const w, myRemoteWidgets)
     w->setEnabled (isRemote);
 
-  for (QWidget * const w: myAuthWidgets)
+  foreach (QWidget * const w, myAuthWidgets)
```

Comment:
On Mon, Mar 28, 2016 at 3:42 PM, Ivan Shapovalov <notifications@github.com>
wrote:

> In qt/SessionDialog.cc
> <https://github.com/transmission/transmission/pull/7#discussion_r57612631>
> :
>
> >      w->setEnabled (isRemote);
> >
> > -  for (QWidget * const w: myAuthWidgets)
> > +  foreach (QWidget * const w, myAuthWidgets)
>
> Why these hunks? Is Qt's foreach faster than C++11's ranged for, or what?
>

For Qt based containers, the Qt foreach is faster because the C++11 foreach
will create a deep copy of all items on the container
while the Q_FOREACH will only copy it's internal data.
( I need to update this patch series urgently)


> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/transmission/transmission/pull/7/files/3dc612adbe16605ef4551b710851f322bf76bb08#r57612631>
>